script/hassfest/quality_scale.py

```diff
@@ -1322,6 +1343,28 @@ def validate_iqs_file(config: Config, integration: Integration) -> None:
             "quality_scale", f"Invalid {name}: {humanize_error(data, err)}"
         )
 
+    if declared_quality_scale is None:
+        return
+
+    rules_met = set()
+    for rule_name, rule_value in data.get("rules", {}).items():
+        status = rule_value["status"] if isinstance(rule_value, dict) else rule_value
+        if status == "done":
```

Comment:
I think you need to add "exempt" here
```suggestion
        if status in {"done", "exempt"}:
```